service/src/requestSchemas.ts

```diff
@@ -117,15 +121,20 @@ export const IdentifyingParameters = z
   .partial();
 
 export const PackageArgs = IdentifyingParameters.extend({
+  canonicalVersion: z.string(),
   capability: z.string(),
-  'check-system-version': z.string(),
+  contentEndpoint: z.string(),
+  checkCanonicalVersion: z.string(),
   count: stringToNumber,
-  'force-system-version': z.string(),
-  'include-components': stringToBool,
-  'include-dependencies': stringToBool,
+  forceCanonicalVersion: z.string(),
```

Comment:
Do we want a TODO comment for the `canonicalVersion` vs. `system-version` params too?